server/config.go

```diff
@@ -30,12 +29,15 @@ const (
 	PutBlobEncodingVersionFlagName       = "eigenda-put-blob-encoding-version"
 	DisablePointVerificationModeFlagName = "eigenda-disable-point-verification-mode"
 	// Kzg flags
-	G1PathFlagName             = "eigenda-g1-path"
-	G2TauFlagName              = "eigenda-g2-tau-path"
-	CachePathFlagName          = "eigenda-cache-path"
-	MaxBlobLengthFlagName      = "eigenda-max-blob-length"
+	G1PathFlagName        = "eigenda-g1-path"
+	G2TauFlagName         = "eigenda-g2-tau-path"
+	CachePathFlagName     = "eigenda-cache-path"
+	MaxBlobLengthFlagName = "eigenda-max-blob-length"
```

Comment:
Can we use "length" and "size" consistently? "length" is number of symbols from the field (32 bytes per symbol), and "size" is number of bytes